OpenOversight/app/utils.py

```diff
@@ -95,6 +96,18 @@ def allowed_file(filename):
            filename.rsplit('.', 1)[1].lower() in current_app.config['ALLOWED_EXTENSIONS']
 
 
+def requires_conversion(filename):
+    return '.' in filename and \
+        filename.split('.', 1)[1].lower() in current_app.config['REQUIRES_CONV']
+
+
+def convert_image(file_to_upload):
```

Comment:
A test here would also be :100: - it could assert:

1. The original image is deleted
2. The modified image is created
3. The name or path of the modified file is returned (see comment above)